tests/test_bookmarks_cursor.py

```diff
@@ -0,0 +1,128 @@
+import os
+
+import singer
+
+import tap_tester.connections as connections
+import tap_tester.menagerie   as menagerie
+import tap_tester.runner      as runner
+
+from base import TestSquareBaseParent
+
+LOGGER = singer.get_logger()
+
+
+class TestSquareIncrementalReplicationCursor(TestSquareBaseParent.TestSquareBase):
+
+    @staticmethod
+    def name():
+        return "tap_tester_square_incremental_replication_cursor"
+
+    def testable_streams_dynamic(self):
+        return self.dynamic_data_streams().intersection(
+            self.expected_full_table_streams()).difference(
+                self.untestable_streams())
+
+    @staticmethod
+    def testable_streams_static():
+        """ No static streams contain cursor bookmarks. """
+        return set()
+
+    @classmethod
+    def tearDownClass(cls):
+        print("\n\nTEST TEARDOWN\n\n")
+
+    def test_run(self):
+        """Instantiate start date according to the desired data set and run the test"""
+
+        self.START_DATE = self.get_properties().get('start_date')
+
+        print("\n\nTESTING WITH DYNAMIC DATA IN SQUARE_ENVIRONMENT: {}".format(os.getenv('TAP_SQUARE_ENVIRONMENT')))
```

Comment:
This can be dropped I think it's a duplicate.

Reply:
Sounds good, will fix

Reply:
Dropped the print